### PR TITLE
Bug: Fix dissolve delay when dissolving

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

The remaining time until dissolution was wrong.

# Changes

* Add an util that calculated the remaining time.
* Use the new util that calculated the remaining time.

# Tests

* Test the new util.

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
UI:

Screenshot 2022-04-07 at 11 08 52
Screenshot 2022-04-07 at 11 40 05

**Comment by lmuntanerDfinity** [comment]
@peterpeterparker @mstrasinskis please review

**Review by mstrasinskis — APPROVED** [review]
LGTM. Only one question.

File: frontend/svelte/src/tests/lib/utils/neuron.utils.spec.ts [hunk 1]
```diff
@@ -185,6 +186,44 @@ describe("neuron-utils", () => {
...
+        fullNeuron: {
+          ...mockFullNeuron,
+          dissolveState: undefined,
+        },
+      };
+      expect(getDissolvingTimeInSeconds(neuron)).toBeUndefined();
+    });
+
+    it("returns duration from today until dissolving time", () => {
+      const todayInSeconds = BigInt(Math.round(Date.now() / 1000));
```
[the remark this answers is not in the document]
Comment (lmuntanerDfinity) [reply frontend/svelte/src/tests/lib/utils/neuron.utils.spec.ts:211]: I thought about it. Here, I care about the difference, not the actual date.

I have run it many times, and it always passes. It matters that it's more than `now`. Not how much or the specific `now`.

I'm not concerned about the passing of time.

[Jest Timer Mocks](https://jestjs.io/docs/timer-mocks) seem to be more concerned about the passing of time in the test.

**Pull request merged**